vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/LocalizedText.java

```diff
@@ -0,0 +1,62 @@
+package edu.asu.diging.vspace.core.model.impl;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import edu.asu.diging.vspace.core.model.ILocalizedText;
+
+
+@Entity
+public class LocalizedText implements ILocalizedText {
+
+    @Id
+    @GeneratedValue(generator = "localized_text_id_generator")
+    @GenericGenerator(name = "localized_text_id_generator", parameters = @Parameter(name = "prefix", value = "LOCTEXT"), strategy = "edu.asu.diging.vspace.core.data.IdGenerator")
+    private String id;
+
+    @OneToOne
```

Comment:
that looks wrong. one language will have multiple localized texts, not just one.